scripts/ci/connect-test-matrix-generator.js

```diff
@@ -84,8 +84,8 @@ const daily = {
     ],
 };
 
-const legacyCanaryFirmware = {
-    firmwares: ['2.2.0', '2-main'],
```

Comment:
ha! no no no, we still want to test the oldest supported firmware :P

Reply:
Should we make legacy firmware `2.3.0` or keep `2.2.0`?

Reply:
lets legacy test the oldest supported fw

Reply:
so yes, 2.3.0